modules/nw-rfhe-creating-hardware-event.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * networking/using-rfhe.adoc
```

Comment:
This says networking/using-rfhe.adoc but I see it under monitoring/using-rfhe.adoc